torch/quantization/ns/graph_matcher.py

```diff
@@ -87,12 +88,47 @@ def get_non_matchable_modules() -> Set[Callable]:
         torch.quantization.FakeQuantizeBase,
     ])
 
-class _NSGraphMatchableNodesIterator:
+def get_reversed_fusions() -> Set[Tuple[Callable, Callable]]:
+    """
+    Set of potential fusions, in reverse order.  The order is reversed
+    to match how fusion patterns are defined in quantization code.
+    """
+    return set([
+        (F.relu, F.linear),
+    ])
+
+# TODO(future PR): we should see if we can reuse quantization's fusion
```

Comment:
Yes, this is a good idea. In general, would the entire graph_matcher logic be shared with fx quantization passes?

Reply:
that would be a good long term state.  I'd imagine most of the benefits to come from sharing the fusion patterns, the matching logic itself may not generalize.